hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/RatisOverReplicationHandler.java

```diff
@@ -321,4 +325,43 @@ private int createCommands(
     return commandsSent;
   }
 
+  static class EligibleAndReserveReplicas {
```

Comment:
Could you help add a short Javadoc on this class and the attributes for clarify?